src/http/api/v1/exports.ts

```diff
@@ -0,0 +1,333 @@
+  coefficient: 0 | 1;
+  type?: string;
+  success?: boolean;
+  rawScore?: string;
+  threshold?: string;
+};
+
+type PassportScoresMap = {
+  [address: string]: FlatPassportScoreWithCoefficient;
+};
+
+async function exportVoteCoefficientsCSV(db: JsonStorage, round: Round) {
+  const processPassportScores = (
+    scores: FullPassportScore[]
+  ): PassportScoresMap => {
+    return scores.reduce((map, score) => {
+      const address = score.address.toLowerCase();
+      const { evidence, error, ...remainingScore } = score;
+      const coefficient = evidence !== null && evidence.success ? 1 : 0;
+      map[address] = { ...remainingScore, ...evidence, coefficient };
+      return map;
+    }, {} as PassportScoresMap);
+  };
+
+  const [votes, data] = await Promise.all([
+    db.collection(`rounds/${round.id}/votes`).all(),
+    fs.promises.readFile("./data/passport_scores.json", {
+      encoding: "utf8",
+      flag: "r",
+    }),
+  ]);
+
+  const passportScores: FullPassportScore[] = JSON.parse(data);
+  const passportScoresMap = processPassportScores(passportScores);
+
+  const records = votes.map((vote: any) => {
+    const voter = vote.voter.toLowerCase();
+    const combinedVote = {
+      ...vote,
+      ...(passportScoresMap[voter] ?? { coefficient: 0 }),
+    };
+
+    return [
+      combinedVote.id,
+      combinedVote.projectId,
+      combinedVote.applicationId,
+      combinedVote.roundId,
+      combinedVote.token,
+      voter,
+      combinedVote.grantAddress,
+      combinedVote.amount,
+      combinedVote.amountUSD,
+      combinedVote.coefficient,
+      combinedVote.status,
+      combinedVote.last_score_timestamp,
+      combinedVote.type,
+      combinedVote.success,
+      combinedVote.rawScore,
+      combinedVote.threshold,
+    ];
+  });
+
+  const csv = createArrayCsvStringifier({
+    header: [
+      "id",
+      "projectId",
+      "applicationId",
+      "roundId",
+      "token",
+      "voter",
+      "grantAddress",
+      "amount",
+      "amountUSD",
+      "coefficient",
+      "status",
+      "last_score_timestamp",
+      "type",
+      "success",
+      "rawScore",
+      "threshold",
+    ],
+  });
+
+  return csv.getHeaderString()!.concat(csv.stringifyRecords(records));
+}
+
+async function exportRoundCSV(round: Round) {
+  const csv = createObjectCsvStringifier({
+    header: [
+      "id",
+      "amountUSD",
+      "votes",
+      "token",
+      "matchAmount",
+      "matchAmountUSD",
+      "uniqueContributors",
+      "applicationMetaPtr",
+      "applicationMetadata",
+      "metaPtr",
+      "metadata",
+      "applicationsStartTime",
+      "applicationsEndTime",
+      "roundStartTime",
+      "roundEndTime",
+      "createdAtBlock",
+      "updatedAtBlock",
+    ].map((h) => ({
+      id: h,
+      title: h,
+    })),
+  });
+
+  return csv.getHeaderString()!.concat(csv.stringifyRecords([round]));
+}
+
+async function exportApplicationsCSV(db: JsonStorage, round: Round) {
```

Comment:
may be annoyingly pedantic and unnecessary, but this function seems like it is doing a lot - extracting question titles, creating a CSV stringifier, and iterating over applications to build the records. Perhaps we could split this up into smaller chunks e.g.
```
function extractQuestionTitles(apps[]): string[]

and 

function buildApplicationRecord(app, titles, etc): records[]
```
That way `exportApplicationsCSV` is only responsible for exporting the CSV.

```typescript
async function exportApplicationsCSV() {
  // ... load from collection ...
  const questionTitles = extractQuestionTitles(apps)
  // ... csv stringifier ...
  const records = apps.map(app => buildApplicationRecord(app, titles, etc)
}
  return csv.getHeaderString() + csv.stringifyRecords(records);

```

if that makes sense at all